pkg/pool-linear/test/LinearPool.test.ts

```diff
@@ -625,8 +625,8 @@ describe('LinearPool', function () {
     });
 
     const itAdaptsTheScalingFactorsCorrectly = () => {
-      const expectedBptScalingFactor = fp(1);
-      const expectedMainTokenScalingFactor = fp(1);
+      const expectedBptScalingFactor = FP_SCALING_FACTOR;
+      const expectedMainTokenScalingFactor = FP_SCALING_FACTOR;
```

Comment:
I'm not a fan of this as it seems to be overloading what `SCALING_FACTOR` means in `FP_SCALING_FACTOR`.

`FP_SCALING_FACTOR` = "Scaling factor to convert a number into a fixed point number. (This is always 1e18)
`expectedBptScalingFactor` = "Scaling factor to convert a token to 18 decimals represented as fixed point number` (This is often 1e18 but sometimes not).

I'd change this to be `FP_ONE` as then it's clear that the scaling factor is 1 but we're showing it in a FP way.